docs/pages/admin-guides/teleport-policy/integrations/crown-jewels.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: See permission changes with Teleport Policy Crown Jewels
+description: Describes how to use Teleport Policy Crown Jewels to see permission changes in Teleport.
+---
+
+With Teleport Policy's Access Graph, you can see change to users and resources permissions.
```

Comment:
Do we need this first sentence? Feels like the first sentence should mention crown jewels in some way, and this one doesnt.